web/src/routes/(user)/utilities/duplicates/[[photos=photos]]/[[assetId=id]]/+page.svelte

```diff
@@ -11,42 +11,124 @@
   import { deleteAssets, updateAssets } from '@immich/sdk';
   import { t } from 'svelte-i18n';
   import type { PageData } from './$types';
+  import { suggestDuplicateByFileSize } from '$lib/utils';
+  import LinkButton from '$lib/components/elements/buttons/link-button.svelte';
+  import { mdiCheckOutline, mdiTrashCanOutline } from '@mdi/js';
+  import Icon from '$lib/components/elements/icon.svelte';
 
   export let data: PageData;
 
-  const handleResolve = async (duplicateId: string, duplicateAssetIds: string[], trashIds: string[]) => {
-    try {
-      if (!$featureFlags.trash && trashIds.length > 0) {
-        const isConfirmed = await dialogController.show({
-          prompt: $t('delete_duplicates_confirmation'),
-          confirmText: $t('permanently_delete'),
-        });
-
-        if (!isConfirmed) {
-          return;
-        }
-      }
+  $: hasDuplicates = data.duplicates.length > 0;
 
-      await updateAssets({ assetBulkUpdateDto: { ids: duplicateAssetIds, duplicateId: null } });
-      await deleteAssets({ assetBulkDeleteDto: { ids: trashIds, force: !$featureFlags.trash } });
-
-      data.duplicates = data.duplicates.filter((duplicate) => duplicate.duplicateId !== duplicateId);
+  const withConfirmation = async (callback: () => Promise<void>, prompt?: string, confirmText?: string) => {
+    if (prompt && confirmText) {
+      const isConfirmed = await dialogController.show({
+        prompt: $t(prompt),
+        confirmText: $t(confirmText),
+      });
 
-      if (trashIds.length === 0) {
+      if (!isConfirmed) {
         return;
       }
+    }
 
-      notificationController.show({
-        message: $t('assets_moved_to_trash', { values: { count: trashIds.length } }),
-        type: NotificationType.Info,
-      });
+    try {
+      return await callback();
     } catch (error) {
       handleError(error, $t('errors.unable_to_resolve_duplicate'));
     }
   };
+
+  const deletedNotification = (trashedCount: number) => {
+    if (!trashedCount) {
+      return;
+    }
+
+    notificationController.show({
+      message: $t($featureFlags.trash ? 'assets_moved_to_trash' : 'assets_deleted', {
```

Comment:
`assets_deleted` is missing

Reply:
Good catch! I renamed it to `permanently_deleted_assets` in the JSON but forgot to change it here.